src/web/components/Core/SelectAllCheckbox.scss

```diff
@@ -0,0 +1,21 @@
+.select-all-checkbox {
+  background-color: var(--theme-radio-background) !important;
```

Comment:
Try to avoid using !important - sometimes it's needed, but you might just need to up the specificity of the selector. With SCSS sometimes the best way is to just nest the whole thing inside a selector for the class of the component - see e.g. Dialog.scss and its use of `.dialog-container`.

Reply:
I've updated all scss invovled in this PR. will udpate the rest when working on styling change ticket